config/routes.rb

```diff
@@ -1,19 +1,43 @@
 Rails.application.routes.draw do
+  get 'sessions/new'
+  get 'sessions/create'
+  # namespace :api do
+  #   namespace :v1 do
+  #     get 'posts/index'
+  #   end
+  # end
+  # namespace :api do
+  #   namespace :v1 do
+  #     get 'users/index'
+  #   end
+  # end
```

Comment:
- [ ] For best practices and clean code, kindly ensure that you remove all the commented and unused code lines from your code base